app/modules/loan/cruds_loan.py

```diff
@@ -190,13 +190,11 @@ async def update_loan(
 
 async def update_loan_returned_status(
     loan_id: str,
-    returned: bool,
     db: AsyncSession,
+    values: dict[str, object],
```

Comment:
Maybe it would be cleaner to pass `returned` and `returned_date` independently as function arguments

Reply:
fixed